### PR TITLE
macOS support

### LINKED ISSUE #25: Mac support
👼

### PR DESCRIPTION
**Pull request by tbaranes**
Close #25 
Device for macOS is exactly the same as iOS, I just removed the helpers methods (maybe others dedicated to mac will come later), and I changed a bit `version`. Since almost each mac can have 2-3 different screen size, it's annoying to test them one by one.

Also, I didn't update the README, I prefer to let you do it since it can be a big change. Otherwise, everything is up to date (including the podspec) :)

### CODE REVIEW

**Comment by tbaranes** [comment]
Rebased against master and migrate the macOS code to swift3

**Comment by tbaranes** [comment]
Conflicts solved 🤘

**Comment by Ekhoo** [comment]
Not sure lol :)

**Comment by tbaranes** [comment]
Rebased! 🙃

**Comment by Ekhoo** [comment]
lol thank you :)

**Comment by Ekhoo** [comment]
Thank you so much for your support

**Comment by tbaranes** [comment]
Happy to contribute, let me know if you need more helps (even to maintain the project) 👍

**Comment by Ekhoo** [comment]
Français en plus :)

**Comment by tbaranes** [comment]
haha 🇫🇷

File: Source/Size.swift [hunk 1]
```diff
@@ -6,15 +6,27 @@
 //  Copyright © 2015 Ekhoo. All rights reserved.
 //
 
-public enum Size: Int, Comparable {
```
Comment (Ekhoo) [inline Source/Size.swift]: Why do you remove the Comparable protocol ?

Comment (tbaranes) [inline Source/Size.swift]: Probably a failed while resolving the conflicts 😑

**Pull request merged**